### ISSUE TITLE
Android 4.3 Remount issue and a small logic error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Kang29/roottools#46]
```
Just a small logic error in 
http://code.google.com/p/roottools/source/browse/trunk/Developmental/RootTools_s
dk3_generic/src/com/stericson/RootTools/internal/RootToolsInternalMethods.java

I think line 1214 should be:
if (!line.contains("no such tool")) {
and not:
if (line.contains("no such tool")) {
as it currently is

Also as the summary said, it seems like remount method does not work on Android 
4.3.

I'm using RootTools 3.1beta with Nexus 7 4.3 BTW.
```

Original issue reported on code.google.com by `thuanb...@gmail.com` on 31 Jul 2013 at 2:54

**Comment by GoogleCodeExporter** [comment]
```
Fixed with RootTools 3.2

Thanks for reporting the bugs!

https://code.google.com/p/roottools/downloads/detail?name=RootTools.jar
```

Original comment by `Stericso...@gmail.com` on 31 Jul 2013 at 6:02

**Comment by GoogleCodeExporter** [comment]
Original comment by `Stericso...@gmail.com` on 31 Jul 2013 at 6:02
- Changed state: **Fixed**
